src/components/editor/tabs.ts

```diff
@@ -80,8 +80,8 @@ class Tabs extends LitElement {
         </button>
         ${when(this.closeableTabs, () => html`
         <button @click = ${() => this.closeFile(file)} aria-label="close">
-          x
-          </button>
+          <svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 384 512"><path d="M376.6 84.5c11.3-13.6 9.5-33.8-4.1-45.1s-33.8-9.5-45.1 4.1L192 206 56.6 43.5C45.3 29.9 25.1 28.1 11.5 39.4S-3.9 70.9 7.4 84.5L150.3 256 7.4 427.5c-11.3 13.6-9.5 33.8 4.1 45.1s33.8 9.5 45.1-4.1L192 306 327.4 468.5c11.3 13.6 31.5 15.4 45.1 4.1s15.4-31.5 4.1-45.1L233.7 256 376.6 84.5z"/></svg>
```

Comment:
Thanks for this!

I'm thinking it might be worth moving the svg template into a separate file and importing it. Can you create a new `icons` directory and add this to `close.js`. 

Then you can export it using a Lit template like the following:

```
import { svg } from 'lit'
export const closeIcon = svg` /** SVG in here */  `
```

and then in the `tabs.ts` file, you can use it like the following:

```
<button>
  {closeIcon}
</button>
```

I believe something along these lines should work :) 

Thanks again for taking the time. 

If possible, are you able to provide a screen shot in the PR description?

Reply:
Created a `icons` directory in the `components ` folder and added  `close.ts`  to it

![image](https://github.com/andrico1234/sandpack-lit/assets/56124154/ebf61f0e-bcb9-4428-ba9a-1c7552c4287c)

Imported `closeIcon` to `tabs.ts` and used it in the button

![image](https://github.com/andrico1234/sandpack-lit/assets/56124154/04e6388e-2de3-4879-983b-d2fdafbbe7e2)

![image](https://github.com/andrico1234/sandpack-lit/assets/56124154/26df064c-031f-48e1-94fe-23387348ec38)

Reply:
Great stuff, 

I should've been more clear, but I meant a screenshot of the icon in rendered in the page 😄

I'll test it locally myself during the next couple of days though.

Thanks again for taking this on

Reply:
oh right 😅